packages/ember-ui/addon/components/table-generic.hbs

```diff
@@ -0,0 +1,32 @@
+{{yield
+  (hash
+    SearchBar=(component
+      "table-generic/search-bar"
+      onSearch=this.onSearch
+      label=(t "global.search")
+      placeholder=(t "global.search")
+    )
+  )
+}}
+<YetiTable
+  class="tableYeti w-full relative"
```

Comment:
les classes CSS doivent sûrement être modifées. Vu qu'elles peuvent être override à l'appel de la table. Un sélecteur de base du genre tpk-table-generic peut-être mis pour avoir un CSS drop-in.